specs/lib/removePipelineExtrasSpec.js

```diff
@@ -1,13 +1,18 @@
 'use strict';
 var fs = require('fs');
+var defined = require('cesium').defined;

```

Comment:
Coding guidelines request that this be written as 

```javascript
var Cesium = require('cesium');
var defined = Cesium.defined;
```

Plus, the `var removePipelineExtras = require('../../lib/removePipelineExtras');` should be moved up to be with the other requires, and defined should be after that.